crates/flow-pkt/release.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# This script is used to release the IANA information elements and protocol numbers as xml files
+set -eux
+
+# Get the current date
+current_date=$(date '+%Y-%m-%d')
+
+# Load the iana information elements from IANA registry as xml
+curl https://www.iana.org/assignments/ipfix/ipfix.xml > iana_ipfix_information_elements.xml
+
+# Insert the comment with the current date after the first line
+{
+  head -n 1 iana_ipfix_information_elements.xml  # Print the first line (the XML declaration)
+  echo "<!-- Date of download: $current_date -->"  # Print the comment with the current date
+  tail -n +2 iana_ipfix_information_elements.xml   # Print the rest of the file starting from line 2
+} > temp.xml && mv temp.xml iana_ipfix_information_elements.xml
+
```

Comment:
Hi Uwe,

Thank you for your input! I made some changes based on your suggestions. Now, the result of the curl command is piped directly into the temp file, which makes the script more concise. We still need a second temporary file for the operation to be successful, though, because using just temp.xml causes the file to be truncated before it can be written correctly. That said, the script is cleaner now. We just need to remove temp.xml at the end, and we’ve also eliminated the need for the move command.

Regarding sed: In the initial draft, I used sed, but I opted for simpler, more straightforward operations instead. The reason is that sed can cause issues on macOS, particularly because the version of sed on macOS differs slightly from the one on Linux.
If I had used sed, I would have had to add a conditional check for the operating system, making the script more complex. Given that I opted for simpler alternatives (such as using basic redirection and temporary files), I decided not to complicate things further by handling this sed discrepancy.

I hope that clarifies things! Thanks again Uwe for reviewing the Pull Request and for your helpful input.